### PR TITLE
tell module reworked (previously mail module)

### PR DESCRIPTION
**Pull request by Nuolong**
Addresses discussion on #18.

### CODE REVIEW

File: src/bobbit/modules/tell.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            MAILBOX[tell_message[0]] = [tell_message[0] + ": " + ' '.join(tell_message[1:])]
+
+    if message.nick in MAILBOX:
+        to_tell = [message.with_body(f'Tell from {msg}') for msg in MAILBOX[message.nick]]
+        del MAILBOX[message.nick]
+        return to_tell
+
+# Command
+
+async def tell(bot, message):
```
Comment (pbui) [inline src/bobbit/modules/tell.py]: Can we combine `tell` and `tell_handle` ?

**Comment by Nuolong** [comment]
Changes applied!

File: src/bobbit/modules/tell.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+the next time they send a message).
+'''
+
+MAILBOX = {}
+
+# Command
+
+async def tell(bot, message):
+    if message.body.startswith("!tell"):
+        tell_message = message.body[5:].split()
```
Comment (pbui) [inline src/bobbit/modules/tell.py]: Let's do:

```
target, *body = message.body[5:].split()
```

File: src/bobbit/modules/tell.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        if tell_message[0] in MAILBOX:
+            MAILBOX[tell_message[0]].append(tell_message[0] + ": " + ' '.join(tell_message[1:]))
+        else:
+            MAILBOX[tell_message[0]] = [tell_message[0] + ": " + ' '.join(tell_message[1:])]
+
+    if message.nick in MAILBOX:
+        to_tell = [message.with_body(f'Tell from {msg}') for msg in MAILBOX[message.nick]]
```
Comment (pbui) [inline src/bobbit/modules/tell.py]: Let's make this a *notice*:

``` 
[message.copy(body=f'Tell from {msg}', channel=None, notice=True) for msg in MAILBOX[message.nick]]
```

File: src/bobbit/modules/tell.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+'''
+
+MAILBOX = {}
+
+# Command
+
+async def tell(bot, message):
+    if message.body.startswith("!tell"):
+        tell_message = message.body[5:].split()
+        if tell_message[0] in MAILBOX:
```
Comment (pbui) [inline src/bobbit/modules/tell.py]: If we make `MAILBOX` a `defaultdict`, we can just do an append, no need for the `if`

**Comment by Nuolong** [comment]
Changes applied!

**Comment by pbui** [comment]
@Nuolong Thank you for your contribution.

**Pull request merged**